### PR TITLE
Add default margins to pages with layouts for printing.

### PR DESCRIPTION
**Pull request by velveret**
Now that our print stylings are removing margins and assuming docs have their own margin, we need to add a margin to our created docs.

### CODE REVIEW

File: dist/ui/czi-editor.css [hunk 1]
```diff
@@ -104,6 +104,7 @@
 @media only print {
   .ProseMirror[data-layout],
   .ProseMirror .czi-editor-body {
+    margin: 15.5mm !important;
     min-height: 0;
     padding: auto !important;
```
Comment (hedgerwang) [inline dist/ui/czi-editor.css]: I think the right fix is to remove `padding: auto !important;`

Comment (velveret) [reply dist/ui/czi-editor.css]: I just tested removing that and it doesn't fix the problem.

**Comment by velveret** [comment]
Closing out for now since it interacts with https://github.com/chanzuckerberg/czi-prosemirror/pull/26

**Comment by velveret** [comment]
Reopening since I'm reverting https://github.com/chanzuckerberg/czi-prosemirror/pull/26 in https://github.com/chanzuckerberg/czi-prosemirror/pull/30

**Comment by hedgerwang** [comment]
I've tried this and it seems work
https://github.com/chanzuckerberg/czi-prosemirror/pull/33

**Pull request closed without merging**